### DISCUSSION TITLE
Calling model find with relation in a component causes undefined offset 1

### DISCUSSION THREAD

**Discussion by monzer15** [livewire/livewire#3088] [7. Bugs (archived)]
### Exact steps to reproduce

    public $session;
    public $participants;
    public $students;
    public function mount($session_id)
    {
        $this->session = Session::with(['exam', 'results', 'sessions_students_lookups.student', 'createdBy'])->findOrFail($session_id);
        $this->students = Student::all();
        $this->participants = collect();
    }

    public function render()
    {
        return view('livewire.dashboard.sessions.details')
            ->extends('dashboard.layouts.app')
            ->section('content');
    }

### Note

dumping $this->session works but still throwing undefined offset 1

### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: [e.g. 2.3.6]
- Laravel version: [e.g. 8.0]
- Browser: Chrome

**Comment by joshhanley** [comment]
@monzer15 can you share a screenshot of the error and share your blade view for this component?
